nixos/modules/config/pulseaudio.nix

```diff
@@ -158,7 +212,7 @@ in {
           wantedBy = [ "default.target" ];
           serviceConfig = {
             Type = "notify";
-            ExecStart = "${getBin cfg.package}/bin/pulseaudio --daemonize=no";
+            ExecStart = "${getBin overriddenPackage}/bin/pulseaudio --daemonize=no";
```

Comment:
Because it's repeated three times? If you think it will be changed often, then yes, else no (because it's quite readable as is).